core/templates/dev/head/css/oppia.css

```diff
@@ -3949,7 +3958,7 @@ md-card.preview-conversation-skin-supplemental-card {
 
 .oppia-activity-summary-tile .metrics {
   height: 30px;
-  margin: 0 12px;
+  margin: 10px 12px 0;
```

Comment:
@markhalpin -- are we happy with the 10px of margin here, or would we prefer something slightly smaller?

10 / 5 / 0:
<img width="215" alt="screen shot 2017-03-04 at 2 17 12 pm" src="https://cloud.githubusercontent.com/assets/1312199/23581636/78a0b396-00e5-11e7-9b18-2a1f7b00d899.png"><img width="211" alt="screen shot 2017-03-04 at 2 19 16 pm" src="https://cloud.githubusercontent.com/assets/1312199/23581642/9b7ba7cc-00e5-11e7-932e-e035f6f604e8.png"><img width="212" alt="screen shot 2017-03-04 at 2 19 23 pm" src="https://cloud.githubusercontent.com/assets/1312199/23581653/cfc9313e-00e5-11e7-9c5c-fa0a000ba960.png">